session3/src/PersonDemo.java

```diff
@@ -0,0 +1,25 @@
+public class PersonDemo {
+
+    public static void main(String[] args) {
+        Person ionutz = new   ();
```

Comment:
this doesn't compile because you are missing the initialization object. I think you wanted `new Person();` here as well.